api/envoy/config/core/v3/protocol.proto

```diff
@@ -55,6 +83,18 @@ message QuicProtocolOptions {
   // window size now, so it's also the minimum.
   google.protobuf.UInt32Value initial_connection_window_size = 3
       [(validate.rules).uint32 = {lte: 25165824 gte: 1}];
+
+  // QUIC keep-alive probing packets work differently from HTTP/2 keep-alive PINGs in a sense that the probing packet
```

Comment:
I think it would be clearer to describe the behavior directly, instead of comparing it to h2 keep-alive (in case the reader isn't familiar with h2 keep-alive already), and then at the end maybe have a note of why it is different from h2.